Common/DocxFormat/Source/XlsxFormat/Common.cpp

```diff
@@ -48,6 +48,11 @@ char* gcvt(double x, int n, char* b)
 #define _gcvt gcvt
 #endif
 
+#ifdef __FreeBSD__
+#define _gcvt(x,n,b) sprintf(b, "%.17g", x)
+#endif
+

```

Comment:
whitespace

Reply:
what do you mean?

Reply:
> what do you mean?

extra blank line

Reply:
there's more than a blank line.. I hope it's not the most important thing of this patch..
Should we really care about that?